WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -113,6 +116,8 @@ extension FeatureFlag {
             return "Jetpack Scan"
         case .activityLogFilters:
             return "Jetpack's Activity Log Filters"
+        case .unreadPostCount:
+            return "Unread P2 Posts Count"
```

Comment:
This isn't just for P2s, is it?

Reply:
It's not P2s only, it's for all sites followed in reader. I have P2s on the brain 🙃 . Updated!